acceptance/setup/aio/010_Install.rb

```diff
@@ -12,6 +12,20 @@ def install_from_local(hosts, package)
       scp_to(agent, package, tmpdir)
     end
 
+   # Move the openssl libs package to a newer version on redhat platforms
+   use_system_openssl = ENV['USE_SYSTEM_OPENSSL']
+
+   if use_system_openssl && agent[:platform].match(/(?:el-7|redhat-7)/)
```

Comment:
Repeated 3 times, seems worth pulling this whole thing out into a helper function.